diff --git a/node_modules/@openshift-console/dynamic-plugin-sdk/lib/utils/fetch/console-fetch-utils.js b/node_modules/@openshift-console/dynamic-plugin-sdk/lib/utils/fetch/console-fetch-utils.js
index 9187456..4edf1a4 100644
--- a/node_modules/@openshift-console/dynamic-plugin-sdk/lib/utils/fetch/console-fetch-utils.js
+++ b/node_modules/@openshift-console/dynamic-plugin-sdk/lib/utils/fetch/console-fetch-utils.js
@@ -131,20 +131,6 @@ export const validateStatus = async (response, url, method, retry) => {
     if (retry && response.status === 429) {
         throw new RetryError();
     }
-    if (response.status === 401 && shouldLogout(url)) {
-        const next = window.location.pathname + window.location.search + window.location.hash;
-        // We can't use regular import from outside this package, so a dynamic import is required
-        // This also breaks a nasty cycle - authSvc.logout calls coFetch (which calls validateStatus)
-        import('@console/internal/module/auth')
-            .then((m) => m.authSvc)
-            .then((authSvc) => {
-            authSvc.logout(next);
-        })
-            .catch((e) => {
-            // eslint-disable-next-line no-console
-            console.error('Error during logout after 401 response', e);
-        });
-    }
     const contentType = response.headers.get('content-type');
     if (!contentType || contentType.indexOf('json') === -1) {
         throw new HttpError(response.statusText, response.status, response);
